papers/drivendata/main.md

```diff
@@ -434,6 +450,10 @@ Complementary ECDF of F1 score for label classes in the holdout set with at leas
 Precision and recall values for a selection of label classes. The top half are the top 10 label classes ranked by F1 score, while the bottom half are the bottom 10 label classes ranked by F1 score. The parenthetical annotation gives the number of observations of that class.
 :::
 
+For an indirect comparison[^footnote-image-benchmarking], SpeciesNet, the model used by Wildlife Insights, report a 81.9% accuracy, 80.7 species-weighted-average F1 score, and 54.2 macro-averaged F1 score[@doi:10.1049/cvi2.12318]. This shows that Zamba's pretrained model is in a rough comparable range of performance as other published results.
+
+[^footnote-image-benchmarking]: As discussed for video species classification, it is difficult to report direct comparisons to other tools or study results due to the lack of standard benchmarking procedures and datasets. Model performance on the same dataset can depend on particular implementation details of a given model, like the specific supported class labels. We encourage users to evaluate models on their own data. Additionally, Zamba supports fine-tuning custom models which can often be the best choice for achieving strong performance.
```

Comment:
the main thing that makes eval hard is that performance on the same _labels_ can depend greatly based on how similar the image/videos are to what the model has seen. a duiker heavily occluded in a jungle is much harder than a duiker in an open field. IMO, it's less about the particular implementation details of a given model and more about the high variance of what data of a certain label looks like